dockstore-webservice/src/main/resources/migrations.1.14.0.xml

```diff
@@ -180,4 +180,93 @@
         </addColumn>
         <addForeignKeyConstraint baseColumnNames="notebookid" baseTableName="event" constraintName="fkNotebookId" deferrable="false" initiallyDeferred="false" onDelete="NO ACTION" onUpdate="NO ACTION" referencedColumnNames="id" referencedTableName="notebook"/>
     </changeSet>
+    <changeSet author="ktran (generated)" id="addMetrics">
+        <createSequence cacheSize="1" cycle="false" incrementBy="1" maxValue="9223372036854775807" minValue="1" sequenceName="countmetric_id_seq" startValue="1"/>
+        <createSequence cacheSize="1" cycle="false" incrementBy="1" maxValue="9223372036854775807" minValue="1" sequenceName="statisticmetric_id_seq" startValue="1"/>
+
+        <createTable tableName="execution_status">
+            <column name="id" type="BIGINT">
+                <constraints nullable="false" primaryKey="true" primaryKeyName="executionstatus_pkey"/>
+            </column>
+            <column name="isvalid" type="BOOLEAN">
+                <constraints nullable="false"/>
+            </column>
+            <column name="numberoffailedexecutions" type="INTEGER">
+                <constraints nullable="false"/>
+            </column>
+            <column name="numberofsuccessfulexecutions" type="INTEGER">
+                <constraints nullable="false"/>
+            </column>
+        </createTable>
+        <createTable tableName="execution_status_count">
+            <column name="executionstatusid" type="BIGINT">
+                <constraints nullable="false" primaryKey="true" primaryKeyName="execution_status_count_pkey"/>
+            </column>
+            <column name="count_id" type="BIGINT">
+                <constraints nullable="false"/>
+            </column>
+            <column name="executionstatus" type="INTEGER">
+                <constraints nullable="false" primaryKey="true" primaryKeyName="execution_status_count_pkey"/>
+            </column>
+        </createTable>
+        <addUniqueConstraint columnNames="count_id" constraintName="uk_count_id_execution_status_count" tableName="execution_status_count"/>
```

Comment:
Interesting, we use `uk_` all over. I thought it was a typo for `uc_` (unique constraint), but I guess it's "unique key". :) Nothing to change